apps/etterlatte-saksbehandling-ui/client/src/components/oppgavebenk/filter/filterLocalStorage.ts

```diff
@@ -0,0 +1,13 @@
+import { Filter } from '~components/oppgavebenk/filter/oppgavelistafiltre'
+
+export const FILTER_KEY = 'filter'
+
+export const leggFilterILocalStorage = (filter: Filter) => {
+  localStorage.setItem(FILTER_KEY, JSON.stringify(filter))
+}
+
+export const hentFilterFraLocalStorage = (): Filter => {
+  return JSON.parse(localStorage[FILTER_KEY])
+}
+
+export const eksistererFilterILocalStorage = (): boolean => !!localStorage[FILTER_KEY]
```

Comment:
Kan med fordel slå disse sammen. De brukes utelukkende samtidig så man kan på denne måten forenkle koden en god del. 

```suggestion
export const hentFilterFraLocalStorage = (): Filter => {
  const filter = localStorage[FILTER_KEY]

  if (!!filter) return JSON.parse(filter)
  else return initialFilter()
}
```